docs/scenarios_authoring/BasicNodes.md

```diff
@@ -98,21 +98,18 @@ There can be at most one edge of type `Default`, and it gets all records that do
 
 ![union_window](img/union_window.png)
 
-Union merges multiple branches into one stream. For each incoming branch two parameters are configured:
-- key - it's value should be of type `String`, definex how elements from branches will be matched together
-- value - this is the output value which will be put the field with name the same as branch id
+Union merges multiple branches into one stream. For each incoming branch only one parameter is configured:
+- value - this is the output value which will be put in the result steam
 
-Union node defines new stream which is union of all branches. In this new stream there is only one variable; it's name is defined by 'Output' parameter; it's value is: 
+Union node defines new stream which is union of all branches. This new steam name is defined by 'Output' parameter;
+If you want to distinguish between branches in output stream you can do this using e.g. map as value of branch
```

Comment:
It should be rather "If you want to distinguish between branches in output stream you can e.g. add extra field in value map. Value expression will then look like for branch1: 
```$json
{ branch: "branch1", unifiedField1: <expression>, unifiedFieldN: <expression> }
```
and for branch2:
```$json
{ branch: "branch2", unifiedField1: <expression>, unifiedFieldN: <expression> }
```
After this, you can check the branch using e.g. `#outputVarible.branch == 'branch1'` expression